app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -14,7 +18,7 @@ import { timeParse, timeFormat, timeFormatLocale } from 'd3-time-format';
 import { min, max } from 'd3-array';
 import { d3locale } from 'lib/shared';
 import { easeLinear } from 'd3-ease';
-import { normalizeString } from "../../../gobierto_visualizations/webapp/lib/utils";
+import { normalizeString, createScaleColors } from "../../../gobierto_visualizations/webapp/lib/utils";
```

Comment:
Wait, I forgot this. Totally wrong. You're settling a dependency between a component and a module

Reply:
Everything declared in _lib/shared_ (or lib/whatever) must work ON HIS OWN.

Reply:
Done!

Reply:
No man, don't you realise you're moving just `createScaleColors`? What happens with `normalizeString`? It's the same.

You cannot create a dependency between modules (that's [coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming))), the wrong thing here is about the path: **../../../gobierto_visualizations/webapp/lib/utils**. Think about it. You're in "lib/vue-components". And you're setting some stuff who comes from _gobierto_visualizations_. Tomorrow I wanna create a BeesWarmChart in _gobierto_budgets_.... and it fetches code from an unrelated module.